SPECS/aspnetcore-runtime-3.1/aspnetcore-runtime-3.1.spec

```diff
@@ -0,0 +1,67 @@
+%define         debug_package %{nil}
+Summary:        Microsoft aspnetcore runtime
+Name:           aspnetcore-runtime-3.1
```

Comment:
Do we still need the `-3.1` in the name for this package, since it's separate now?

Reply:
Yeah I think we need to keep the `-3.1` since it corresponds with the existing `dotnet-runtime-3.1`